vm_setup.pl

```diff
@@ -10,8 +10,9 @@
 use IO::Select;
 use String::Random;
 use IPC::Open3;
+use Term::ANSIColor qw(:constants);
```

Comment:
Please use either `$Term::ANSIColor::AUTOLOCAL = 1` or `$Term::ANSIColor::AUTORESET = 1` when using this:

http://search.cpan.org/~rra/Term-ANSIColor-4.06/lib/Term/ANSIColor.pm

At this time the original terminal foreground/background colors are not reset to default after program termination.  When I tested it resulted in using `BOLD WHITE ON_BLACK` after the program ended.